Pipfile

```diff
@@ -9,6 +9,7 @@ protobuf = "==3.20.*"
 bip-utils = "*"
 gql = {extras = ["all"], version = "*"}
 psycopg = "*"
+python-dateutil = "*"
```

Comment:
I didn't see any references in the test, are we actually using this anywhere?

Reply:
This is a remnant of what is used in tests/base.py - the helper function to gather latest block timestamp

Reply:
I don't see it being used in test/base.py in this branch, could it be in another branch?